pkg/api/api.proto

```diff
@@ -212,10 +212,20 @@ service Manager {
             get: "/api/Manager/GetSavedModels"
         };
     };
+    /** 
+     * Validate Suggestion Parameters from Study Job.
```

Comment:
Please explain a suggestion service should return `INVALID_ARGUMENT` error when the parameter is invalid.